src/current/v24.3/manage-logical-data-replication.md

```diff
@@ -0,0 +1,195 @@
+- [Schema changes](#schema-changes): The tables that are part of the LDR job may require schema changes, which need to be coordinated manually. There are some schema changes that are supported while LDR is active.
+- [Jobs](#jobs-and-ldr): Other CockroachDB jobs can operate on clusters running LDR [jobs]({% link {{ page.version.version }}/show-jobs.md %}). You may want to consider where you start and how you manage [backups]({% link {{ page.version.version }}/backup-and-restore-overview.md %}), [changefeeds]({% link {{ page.version.version }}/change-data-capture-overview.md %}), [row-level TTL]({% link {{ page.version.version }}/row-level-ttl.md %}), and so on when you're running LDR.
+
+## Conflict resolution
+
+Conflicts in LDR are detected when there is either:
+
+- An `UPDATE` operation replicated to the destination cluster.
+- A cross-cluster write occurs, which means both clusters are writing to the same key. For example, if the LDR stream attempts to apply a row to the destination cluster where the existing row on the destination was not written by the LDR stream.
+
+LDR uses _last write wins (LWW)_ conflict resolution based on the [MVCC timestamp]({% link {{ page.version.version }}/architecture/storage-layer.md %}#mvcc) of the replicating write. LDR will resolve conflicts by inserting the row with the latest MVCC timestamp. When a conflict cannot apply, due to violating a foreign key constraint or schema constraint, it will be retried for up to a minute and then put in the [DLQ](#dead-letter-queue-dlq) if it could not be resolved. 
+
+### Dead letter queue (DLQ)
+
+When the LDR job starts, it will create a DLQ table with each replicating table so that unresolved conflicts can be tracked. The DLQ will contain the writes that LDR cannot apply after the retry period, which could occur if:
+
+- The destination table was dropped.
+- The destination cluster is unavailable.
+- Tables schemas do not match.
+
+In `validated` mode, rows are also sent to the DLQ when:
+
+- [Foreign key]({% link {{ page.version.version }}/foreign-key.md %}) dependencies are not met where there are foreign key constraints in the schema.
+- Unique indexes and other constraints are not met.
+
+{{site.data.alerts.callout_info}}
+LDR will not pause when the writes are sent to the DLQ, you must manage the DLQ manually.
+{{site.data.alerts.end}}
+
+To manage the DLQ, you can evaluate entries in the `incoming_row` column and apply the row manually to another table with SQL statements.
```

Comment:
Any way we could prep a query to help with this before we publish this doc?

Reply:
Yes @msbutler @dt can u help